sdk/metric/controller/push/push.go

```diff
@@ -109,21 +108,6 @@ func (c *Controller) SetErrorHandler(errorHandler sdk.ErrorHandler) {
 	c.accumulator.SetErrorHandler(errorHandler)
 }
 
-// Meter returns a named Meter, satisifying the metric.Provider
-// interface.
-func (c *Controller) Meter(name string) metric.Meter {
```

Comment:
I think I'm good with removing this, but wondering if we thought about changing this to wrap the `Provider` field.

I think I'm in favor or removing as it makes the `Provider` more explicit, but just wondering if there were other thoughts.

Reply:
Instead, I added a `Provider()` getter to be even more explicit. I think it helps readability to see `pusher.Provider().Meter(...)` instead of `pusher.Meter(...)`.